docs/getting-started/explore-analysis.md

```diff
@@ -23,25 +19,24 @@ This includes:
 
 ## Run existing modules
 
-Each existing module will have it's own folder in `analyses`.
-All instructions and files needed to run a given module will be contained in the module's folder.
+Each existing module has its own folder in the repository's [`analyses` folder](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses).
+See our documentation on [analysis modules](../contributing-to-analyses/analysis-modules/index.md) to learn more about how analyses are structured.
 
-- To [run existing modules](../contributing-to-analyses/analysis-modules/running-a-module.md), please refer to the `README.md` found in the root directory of that module.
-- The `README.md` will contain a description of the module, instructions to run the module, the list of input and output files, and any additional software or computing resources you may need.
+All instructions and files (except [data](#obtain-data-to-run-a-module)) needed to run a given module are provided the module's folder.
 
-See [analysis module](../contributing-to-analyses/analysis-modules/index.md) to learn more about how analyses are structured.
+- To [run existing modules](../contributing-to-analyses/analysis-modules/running-a-module.md), please refer to the `README.md` found in the root directory of that module.
+    - The `README.md` will contain a description of the module, instructions to run the module, the list of input and output files, and any additional software or computing resources you may need.
+    - The `README.md` will also contain information about the input data that the module takes.
 
-## Accessing data for modules
+## Obtain data to run a module
 
-The specific requirements for input data will be listed in the `README.md` for each module.
-For modules that use data obtained from the ScPCA Portal as input, data can be directly downloaded from the [ScPCA Portal](https://scpca.alexslemonade.org/).
+For modules that use data obtained from the ScPCA Portal as input, you can directly download data from the [ScPCA Portal](https://scpca.alexslemonade.org/).
 
-Additionally, you can use the [`simulate-sce`](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/simulate-sce) analysis module to create simulated data.
+Alternatively, you can use the [`simulate-sce`](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/simulate-sce) analysis module to create simulated data:
```

Comment:
This isn't an alternative since they need to download data first. So I would change this back.